essence-of-live-coding/test/Cell/Util.hs

```diff
@@ -117,4 +118,27 @@ test = testGroup "Utility unit tests"
     , output1 = [Nothing, Just 2]
     , output2 = [Just 3, Just 4]
     }
+  , testProperty "resampleListPar works as expected"
+    $ forAll (vector 100) $ \(inputs :: [(Int, Int)]) -> let 
```

Comment:
This can be a `CellSimulation` as well, it works together with quickcheck.

Reply:
changed.